### DISCUSSION TITLE
Quasar framework incomplete build on mac

### DISCUSSION THREAD

**Discussion by madmax17** [quasarframework/quasar#12846] [CLI - Capacitor / Cordova modes]
My app build on the quasar framework seems to be incomplete, works fine on android but on mac I get incomplete files which I can't open in Xcode.

**Comment by ajmas** [comment]
What errors are you getting? Have you tried removing the iOS target and re-attempting?

**Reply by madmax17** [reply]
I can't see any errors, I don't get the xcodeproj file which I think is needed to open the project in xcode.

**Comment by ajmas** [comment]
Can you provide more information, such as quasar version and what commands you are typing in the cli?

**Reply by madmax17** [reply]
ver: "quasar": "^1.14.0"

quasar build -m cordova -T ios
